INSTALL.md

```diff
@@ -1,112 +1,25 @@
-Installation of PyWBEM Client
-=============================
+Installation of pywbem
+======================
 
-The PyWBEM Client can be installed quite easily by running its standard Python
-setup script (`setup.py`) with the `install` command, or by using `pip install`
-(which also invokes the setup script).
+To install the latest released version of pywbem on Linux:
 
-As of PyWBEM Client v0.8, the setup script has support for installing its
-prerequisites. This includes installing Python packages and OS-level packages,
-and it includes the usual install mode and development mode.
+* Install the prerequisite OS-level packages:
 
-OS-level prerequisites are installed using new setup.py commands `install_os`
-(for the usual install mode) and `develop_os` (for development mode). These
-commands perform the installation for a number of well-known Linux
-distributions, using `sudo` under the covers (so your userid needs to be
-authorized for `sudo`, if you run these commands). For other Linux
-distributions and operating systems, these setup.py commands just display the
-names of the OS-level
-packages that would be needed on RHEL, leaving it to the user to
-translate the package names to the actual system, and to establish the
-prerequisites. This approach is compatible with PyPI because `pip install`
-invokes `setup.py install` but not the new commands. It is also compatible
-with packaging PyWBEM into OS-level Python packages, for the same reason.
-This approach is also compatible with
-[virtual Python environments](https://docs.python-guide.org/en/latest/dev/virtualenvs/)
-because `sudo` is invoked under the covers for installing the OS-level
-packages, so you can still invoke setup.py without sudo for targeting the
-current virtual Python environment.
+  * Download the
+    [pywbem_os_setup.sh](https://pywbem.readthedocs.io/en/stable/_downloads/pywbem_os_setup.sh)
```

Comment:
Only covers linux and we really do need to cover windows.

Reply:
The README also just covers Linux. This is likely to be the most common use case these days.

In both cases (README and INSTALL.md), we are referencing the install section and provide a link that takes you to exactly the install section.

Reply:
OK by me